docs/tutorials/docker-compose.yml

```diff
@@ -68,9 +68,9 @@ services:
                        cub sr-ready schema-registry 8081 20 && \
                        echo Waiting a few seconds for topic creation to finish... && \
                        sleep 2 && \
-                       java -jar /usr/share/java/ksql-examples/ksql-examples-4.1.1-SNAPSHOT-standalone.jar
+                       /usr/bin/ksql-datagen
```

Comment:
@apurvam : if this can now be invoked ` ksql-datagen`, do you want to use that here?

Reply:
We could. But this is consistent with the usages of the other commands in this file.

Reply:
@apurvam : none of them should need the path any more.